cmd/rokka/cli/massupload/file_windows.go

```diff
@@ -0,0 +1,24 @@
+package massupload
+
+// fixpath returns an absolute path on the current OS, so we can open long
+// file names. See Restic file_windows.go
+func fixpath(name string) string {
+	abspath, err := filepath.Abs(name)
+	if err == nil {
+		// Check if \\?\UNC\ already exist
+		if strings.HasPrefix(abspath, `\\?\UNC\`) {
+			return abspath
+		}
+		// Check if \\?\ already exist
+		if strings.HasPrefix(abspath, `\\?\`) {
+			return abspath
+		}
+		// Check if path starts with \\
+		if strings.HasPrefix(abspath, `\\`) {
+			return strings.Replace(abspath, `\\`, `\\?\UNC\`, 1)
+		}
+		// Normal path
+		return `\\?\` + abspath
+	}
+	return name
```

Comment:
aren't functions for that already in the standard lib?

Reply:
Not sure, here a reference to that issue: https://blog.klauspost.com/long-windows-paths-unc-paths-in-go/